lte/gateway/c/core/oai/tasks/sgw_s8/sgw_s8_s11_handlers.h

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 #include "lte/gateway/c/core/oai/lib/itti/intertask_interface.h"
 #include "lte/gateway/c/core/oai/include/spgw_types.h"
 
+#define CP_TEID 1
+#define UP_TEID 2
```

Comment:
leave a blank line after. Any reason for not defining this as enum?

Reply:
I used macro, as only 2 parameters are there